kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/user/UserLibrary.java

```diff
@@ -75,7 +75,7 @@
  */
```

Comment:
Please update license header here from `* Copyright (c) 2017, Gluon and/or its affiliates.` to `* Copyright (c) 2017, 2022, Gluon and/or its affiliates.`.